### ISSUE TITLE
weird query errors while backfilling

### ISSUE THREAD

**Issue by rubycut** [influxdata/influxdb#6069]
Query doesn't work if it has date few day in the back:

```
> SELECT mean(time_total) AS time_total INTO telegram."biweekly".curl_response_time FROM telegram."default".curl_response_time WHERE time > '2016-03-01' AND time < now() GROUP BY time(15s), env, host, url fill(none)
ERR: too many points in the group by interval. maybe you forgot to specify a where time clause?
> SELECT mean(time_total) AS time_total INTO telegram."biweekly".curl_response_time FROM telegram."default".curl_response_time WHERE time > '2016-03-05' AND time < now() GROUP BY time(15s), env, host, url fill(none)
name: result
------------
time                    written
1970-01-01T00:00:00Z    23034
```

**Comment by jwilder** [comment]
What version are you using?

**Comment by rubycut** [comment]
influxdb_0.10.1-1_amd64.deb

**Comment by jwilder** [comment]
@jsternberg @benbjohnson Is this limit still in place for 0.11?

**Comment by benbjohnson** [comment]
That limit was removed in 0.11.

**Comment by jwilder** [comment]
@rubycut Should be fixed in 0.11.
